### PR TITLE
Updated changelog with newest release

### PR DESCRIPTION
**Pull request by lilosti**
https://jira.itkdev.dk/browse/SUPP0RT-738

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
+<!-- markdownlint-configure-file { "blanks-around-headers": { "lines_below": 0 } } -->
+<!-- markdownlint-configure-file { "blanks-around-lists": false } -->
```
Comment (rimi-itk) [inline CHANGELOG.md:2]: I think we should stick to the already defined Markdown linting rules defined in this project, but add exceptions for duplicated headers in `CHANGELOG.md` (cf. https://github.com/itk-dev/naevnssekretariatet/actions/runs/3427044717/jobs/5709564181).

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+See [keep a changelog](https://keepachangelog.com/en/1.0.0/) for information about writing changes to this log.
+
+## [Unreleased]
+
+## [1.0.1] 9/11 2022
```
Comment (rimi-itk) [inline CHANGELOG.md:14]: I think we should use dates on the form YYYY-MM-DD.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+### Fixed
+- [DEVSUPP-1045](https://jira.itkdev.dk/browse/SUPP0RT-738): 
+Fixed error in PDF creation cause by unescaped characters
+- [DEVSUPP-1047](https://jira.itkdev.dk/browse/SUPP0RT-740): 
+Fixed error in unescaped characters in filename
 
-## [Released]
+## [1.0.0] All released before 8/11 2022
 
 ### Added
```
Comment (rimi-itk) [inline CHANGELOG.md:31]: We need 

```
[Unreleased]: https://github.com/itk-dev/naevnssekretariatet/compare/1.0.1...HEAD
[1.0.1]: https://github.com/itk-dev/naevnssekretariatet/compare/1.0.0...1.0.1
[1.0.0]: https://github.com/itk-dev/naevnssekretariatet/releases/tag/1.0.0
```

at the end of the file (cf. https://keepachangelog.com/en/1.0.0/)

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (lilosti) [reply CHANGELOG.md:2]: hvordan gør man det?

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**